lib/browser.js

```diff
@@ -18,3 +18,4 @@ require('./http/xhr');
 
 if (typeof window !== 'undefined') window.AWS = AWS;
 if (typeof module !== 'undefined') module.exports = AWS;
+if (typeof self !== 'undefined') module.exports = AWS;
```

Comment:
@chrisradek you are right, I blame myself. I fixed and squashed my commits. thank you.